### ISSUE TITLE
"Preview Your Content" Content Discussion

### ISSUE THREAD

**Issue by jaredmorgs** [asciidoctor/asciidoctor.org#449]
http://asciidoctor.org/docs/user-manual/#preview-your-content is currently available to create content for.

Use this issue as a tracker for content suggestions, or feel free to provide PRs referencing this ticket to keep track of contributions.

This content is contained in https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/user-manual.adoc

**Comment by jaredmorgs** [comment]
This is a great section to promote the other OpenDevise Asciidoctor projects that make plugins for editors, browsers and other tooling.

**Comment by jaredmorgs** [comment]
Oh, and don't forget to mention the Gist Asciidoctor viewer that you always remind me of, @mojavelinux . That is a great tool for quick collaboration and I always forget it exists or how to use it properly.

**Comment by mojavelinux** [comment]
I totally agree, though this has nothing to do with OpenDevise. We'll simply be promoting tools that make up the Asciidoctor ecosystem, some initiated by OpenDevise, some by others. Who started it doesn't matter. It's just Asciidoctor.

http://gist.asciidoctor.org is an amazing front-end to GitHub and many other repositories and maturing quickly. It even has experimental edit and save support for certain storage locations.
